exonet/exonet.github.io
# Adjust layout to show posts on homepage

## What does it do?
- Replace default homepage with paginated posts.
- Add post-archive page
- Workaround for jekyll3 css and js include bug.

## How to use
When writing posts, optionally an author name and url can be provided in the post metadata. This shows who wrote the post. See the example in "How to test" .

## How to test?
Update your local dependencies. See [Development setup](https://github.com/exonet/exonet.github.io/blob/master/DEV_SETUP.md).

Create a post: `_posts/2016-01-01-test-post.md` with the following content:
```
---
layout: post
title: Test post title
subtitle: Post subtitle
description: Post description
author:
  name: Exonet
  url: http://github.com/exonet
---

Post content. This is a useless post.
```

Build the project with jekyll.
The homepage should display the first 5 posts and the archive should show links to all posts.

## Note
The jekyll3 js and css inclusion is still a problem.
All script includes have to be repeated twice because on 2016-02-01 GitHub pages migrated to jekyll 3.
See bug https://github.com/jekyll/jekyll/issues/4439 and https://github.com/daattali/beautiful-jekyll/issues/33

## Merge restriction
Only merge this after there is at least one post. Otherwise the homepage has nothing to show.
